lib/studies/reply/adaptive_nav.dart

```diff
@@ -916,8 +946,8 @@ class _BottomDrawerDestinations extends StatelessWidget {
               Future.delayed(
                 Duration(
                   milliseconds: GalleryOptions.of(context).timeDilation == 1
-                      ? drawerController.value == 1 ? 300 : 180
-                      : drawerController.value == 1 ? 1500 : 900,
+                      ? drawerController.value == 1 ? 300 : 120
```

Comment:
Should these values automatically scale with timeDilation (and not a constant factor of 5)?

Reply:
I don't think it's necessary since the timeDilation can only be 1 or 5 in the gallery, nothing in between. But by scaling it by timeDilation it could make this if statement look a lot better.

Reply:
done!